src/components/token-avatar.tsx

```diff
@@ -0,0 +1,29 @@
+import { DynamicColorCircle } from '@stacks/ui';
+import { FungibleTokenMetadata } from '@stacks/blockchain-api-client';
+
+import React from 'react';
+import { getAssetNameParts } from '@common/utils';
+import { imageCanonicalUriFromFtMetadata } from '@common/utils/token-utils';
+export function TokenAvatar({
```

Comment:
can you define components with React.FC?
```
interface TokenAvatarProps {}
export const TokenAvatar: React.FC<TokenAvatarProps> = ({}) => ...
```

Reply:
I think React.FC here is not useful, this component doesn't even use any children. I prefer in general to be explicit in the props.

Reply:
what are your thoughts on always defining props as a separate interface on top of the component?

Reply:
you did it already, nice!

Reply:
Yes I like it too, thanks for pointing it out !